common.mk

```diff
@@ -253,6 +253,14 @@ BUILDROOT_TOOLCHAIN=toolchain-aarch$(COMPILE_NS_USER)-legacy
 endif
 endif
 
+#Network Configuration packages( DHCPCD,ETHTOOL,XINETD)
+BR2_PACKAGE_DHCPCD ?= y
+BR2_PACKAGE_ETHTOOL ?= y
+BR2_PACKAGE_XINETD ?= y
+#ssh packages
+BR2_PACKAGE_OPENSSH ?= y
+BR2_PACKAGE_OPENSSH_SERVER ?= y
+BR2_PACKAGE_OPENSSH_KEY_UTILS ?= y
```

Comment:
remove this part of the change (keep only the change in rpi3.mk)

Reply:
ok.